buildLogic/settings.gradle

```diff
@@ -1,7 +1,5 @@
 pluginManagement {
   repositories {
-    mavenCentral()
```

Comment:
I would keep mavenCentral first because gradlePluginPortal uses JCenter with some problems: https://github.com/gradle/gradle/issues/15406

Reply:
Restored - and it looks like you could actually go the other way and leave only `mavenCentral()` here, if you wanted.

Reply:
Nice, yes.